manifests/repo/nodesource.pp

```diff
@@ -59,52 +53,19 @@
       class { '::nodejs::repo::nodesource::yum': }
       contain '::nodejs::repo::nodesource::yum'
 
-      if $::operatingsystemrelease =~ /^5\.(\d+)/ {
-        # On EL 5, EPEL needs to be applied first
-        Class['::epel'] -> Class['::nodejs::repo::nodesource::yum']
-      }
-
     }
     'Linux': {
```

Comment:
I think you can just remove the whole Linux block and allow it to fall back to the generic default.

Reply:
Done